src/stream/src/executor/agg_common.rs

```diff
@@ -50,7 +50,9 @@ pub struct AggExecutorArgs<S: StateStore, E: AggExecutorExtraArgs> {
 
 pub trait AggExecutorExtraArgs {}
 
-pub struct SimpleAggExecutorExtraArgs {}
+pub struct SimpleAggExecutorExtraArgs {
+    pub metrics: Arc<StreamingMetrics>,
```

Comment:
I guess this can be moved to `// agg common things` section of `AggExecutorArgs`